docker-compose.yaml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+services:
+  inv_sig_helper:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    # image: quay.io/invidious/inv-sig-helper:latest
+    command: ["--tcp", "127.0.0.1:12999"]
+    ports:
+      - 127.0.0.1:12999:12999
+    environment:
+      - RUST_LOG=info
+    restart: unless-stopped
+    cap_drop:
+      - ALL
+    read_only: true
+    user: 10001:10001
```

Comment:
Is this really necessary if we are already running it under the same uid?

Reply:
Agreed that it's probably redundant to specify it in `docker-compose.yaml`, but it might still be useful for documentation purposes for the end user